test/acceptance/vector_distances/setup_test.go

```diff
@@ -34,6 +34,8 @@ func Test_GraphQL(t *testing.T) {
 	t.Run("test dot distance", testDot)
 	t.Run("import l2 test data ", addTestDataL2)
 	t.Run("test l2 distance", testL2)
+	t.Run("import hamming test data ", addTestDataHamming)
```

Comment:
Create and delete schema step is missing. To make this `e2e` fully working one needs to create explicitly a class that uses `hamming` as a distance method. In order to do that you need to create a class like that (you can do this in `addTestSchemaOther` method):
```
createObjectClass(t, &models.Class{
		Class:      "Hamming_Class",
		Vectorizer: "none",
		Properties: []*models.Property{
			{
				Name:     "name",
				DataType: []string{"string"},
			},
		},
		VectorIndexConfig: map[string]interface{}{
			"distance": "hamming",
		},
	})
```
and after your `e2e` tests are complete it would be great to delete this class by invoking:
```
deleteObjectClass(t, "Hamming_Class")
```

Reply:
I have made the requested changes, let me know if something more needs to be done 😇